readthedocs/projects/forms.py

```diff
@@ -236,6 +237,46 @@ class Meta(object):
         )
 
 
+class ProjectRelationshipForm(forms.ModelForm):
+
+    """Form to add/update project relationships"""
+
+    parent = forms.CharField(widget=forms.HiddenInput(), required=False)
+
+    class Meta(object):
+        model = ProjectRelationship
+        exclude = []
+
+    def __init__(self, *args, **kwargs):
+        self.project = kwargs.pop('project', None)
+        self.user = kwargs.pop('user', None)
```

Comment:
Should we be checking for the existence of these variables here? Seems like it would throw errors accessing None later on.